apis/kueue/webhooks/workload_webhook.go

```diff
@@ -76,13 +126,31 @@ func setContainersDefaults(containers []corev1.Container) {
 			if c.Resources.Requests == nil {
 				c.Resources.Requests = make(corev1.ResourceList)
 			}
-			for k, v := range c.Resources.Limits {
-				if _, exists := c.Resources.Requests[k]; !exists {
-					c.Resources.Requests[k] = v.DeepCopy()
-				}
-			}
+			c.Resources.Requests = mergeResourceList(c.Resources.Requests, c.Resources.Limits)
+		}
+	}
+}
+
+func setContainersDefaultsFromLimitRanges(containers []corev1.Container, limits []corev1.LimitRangeItem) {
+	for _, ri := range limits {
```

Comment:
can there be more than one Container LimitRange per namespace?

Reply:
Yes, and our "merge" strategy is the same as the one used by  https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/limitranger/admission.go

Reply:
Please reference the relevant code from kubernetes/kubernetes in a comment.